owiddata/owiddata-stats.json

```diff
@@ -1,22 +1,33 @@
 {
-  "owid_RNA_count": 2,
-  "owid_RNA_countries": 163,
-  "owid_RNA_map": "https://github.com/greenelab/covid19-review/raw/ee30031393799c211a140f9be7eefbed12ef0e1b/owiddata/maps/RNA.png",
-  "owid_commit_date": "2022-03-28T10:21:39Z",
-  "owid_commit_date_pretty": "March 28, 2022",
-  "owid_commit_sha": "b4b2b815205521644734392a042c1c443b9f6900",
-  "owid_daily_rate": "1,980 per million",
-  "owid_inactivated_whole_virus_count": 8,
-  "owid_inactivated_whole_virus_countries": 113,
-  "owid_inactivated_whole_virus_map": "https://github.com/greenelab/covid19-review/raw/ee30031393799c211a140f9be7eefbed12ef0e1b/owiddata/maps/inactivated_whole_virus.png",
-  "owid_most_recent_date": "March 27, 2022",
-  "owid_non_replicating_viral_vector_count": 5,
-  "owid_non_replicating_viral_vector_countries": 200,
-  "owid_non_replicating_viral_vector_map": "https://github.com/greenelab/covid19-review/raw/ee30031393799c211a140f9be7eefbed12ef0e1b/owiddata/maps/non_replicating_viral_vector.png",
-  "owid_protein_subunit_count": 9,
-  "owid_protein_subunit_countries": 21,
-  "owid_protein_subunit_map": "https://github.com/greenelab/covid19-review/raw/ee30031393799c211a140f9be7eefbed12ef0e1b/owiddata/maps/protein_subunit.png",
+  "owid_RNA_count": 3,
+  "owid_RNA_countries": 0,
```

Comment:
These country counts are lower than they were before, and sometimes 0.  Do you know what changed?

Reply:
Ahhh! It's a merge issue for the two data sources. I'll add in another function to manually align their nomenclature.